src/components/DpIcon/util/iconVariables.js

```diff
@@ -142,6 +142,14 @@ const baseIcons = {
   lock: {
     path: 'M208 80h-32V56a48 48 0 0 0-96 0v24H48a16 16 0 0 0-16 16v112a16 16 0 0 0 16 16h160a16 16 0 0 0 16-16V96a16 16 0 0 0-16-16ZM96 56a32 32 0 0 1 64 0v24H96Zm112 152H48V96h160v112Z'
   },
+  mapPin: {
+    phosphorName: 'map-pin',
+    path: 'M128 64a40 40 0 1 0 40 40 40 40 0 0 0-40-40Zm0 64a24 24 0 1 1 24-24 24 24 0 0 1-24 24Zm0-112a88.1 88.1 0 0 0-88 88c0 31.4 14.51 64.68 42 96.25a254.19 254.19 0 0 0 41.45 38.3 8 8 0 0 0 9.18 0 254.19 254.19 0 0 0 41.37-38.3c27.45-31.57 42-64.85 42-96.25a88.1 88.1 0 0 0-88-88Zm0 206c-16.53-13-72-60.75-72-118a72 72 0 0 1 144 0c0 57.23-55.47 105-72 118Z'
+  },
+  mapPinSolid: {
+    phosphorName: 'map-pin',
```

Comment:
I like to add the weight so it's clear which variant of the phosphor icon we're using:
`weight: 'fill'`.
But if you're also creating a PR for the vue3 version of demosplan-ui maybe it's not needed.

Reply:
here is [the other PR](https://github.com/demos-europe/demosplan-ui/pull/1182).